src/wms/src/main/java/org/geoserver/wms/map/GetMapKvpRequestReader.java

```diff
@@ -243,12 +243,27 @@ public GetMapRequest read(Object request, Map kvp, Map rawKvp) throws Exception
         // set the raw params used to create the request
         getMap.setRawKvp(rawKvp);
 
+        boolean citeCompliant = wms.getServiceInfo().isCiteCompliant();
+
         // wms 1.3, srs changed to crs
         if (kvp.containsKey("crs")) {
             getMap.setSRS((String) kvp.get("crs"));
+        } else if (citeCompliant) {
```

Comment:
Need to check the version of the protocol, this exception cannot be thrown if the version is any 1.x

Reply:
Done, thanks @aaime